### PR TITLE
Immersive mode

### PR DESCRIPTION
**Pull request by aouerfelli**

### CODE REVIEW

File: app/src/main/java/io/github/gmathi/novellibrary/activity/ReaderPagerActivity.kt [hunk 1]
```diff
@@ -223,23 +248,23 @@ class ReaderPagerActivity : BaseActivity(), ViewPager.OnPageChangeListener, Floa
...
+        val webPage = chapters[index]
+        if (webPage.novelId != -1L && webPage.id != -1L) {
+            dbHelper.updateBookmarkCurrentWebPageId(webPage.novelId, webPage.id)
+        }
+        if (webPage.id != -1L) {
+            webPage.isRead = 1
+            dbHelper.updateWebPageReadStatus(webPage)
+        }
+        return true
     }
```
Comment (gmathi) [inline app/src/main/java/io/github/gmathi/novellibrary/activity/ReaderPagerActivity.kt]: Thanks for the cleanup!

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -326,5 +329,9 @@
     <string name="title_activity_chapter_downloads">ChapterDownloadsActivity</string>
     <string name="delete_chapters">Delete Chapters</string>
 
+    <!-- v0.8 Strings -->
+
+    <string name="immersive_mode" translatable="false">Show Full Screen</string>
+    <string name="immersive_mode_description" translatable="false">Hide the system bars when reading</string>
```
Comment (gmathi) [inline app/src/main/res/values/strings.xml]: Should not be "translatable=false" as these string will be needing translations. For now add them in all the other string files too.

**Comment by aouerfelli** [comment]
Done.
Are you fine with the fallback for devices under KitKat? Didn't want to hack around it but the experience should still be more immersive than before.

**Comment by gmathi** [comment]
Should be fine, but did you do a test-run on devices under KitKat? I want to make it sure, it doesn't crash or behave weirdly!

**Comment by aouerfelli** [comment]
Yeah I always test on API levels 26 and 16, there weren't any problems as
far as I could tell.

**Pull request merged**